lib/OrdinaryDiffEqSymplecticRK/src/algorithms.jl

```diff
@@ -1,6 +1,10 @@
+@doc generic_solver_docstring("DESCRIPTION TBD",
```

Comment:
Quite a bit to still fill in for each symplectic integrators @ChrisRackauckas.

Reply:
@ChrisRackauckas bump
Description for:
VelocityVerlet
VelocityLeapfrog
PseudoVerletLeapfrog
McAte2
McAte3
Ruth3
CandyRoz4
McAte4
CalvoSanz4
Yoshida6
KahanLi6
McAte8
KahanLi8
SofSpa10

Descriptions and references for:
SymplecticEuler
McAte42
McAte5

Reply:
VelocityVerlet, PseudoVerletLeapfrog, and VelocityLeapfrog are just the well known schemes https://en.wikipedia.org/wiki/Verlet_integration in their subforms.

McAte methods and references to a lot of the other 4th order and below are in table 2 from https://www.researchgate.net/publication/230979099_The_Accuracy_of_Symplectic_Integrators https://iopscience.iop.org/article/10.1088/0951-7715/5/2/011

Reply:
https://scicomp.stackexchange.com/questions/29149/what-does-symplectic-mean-in-reference-to-numerical-integrators-and-does-scip should be used in the summary text for the symplectic integrators page.

Reply:
SymbplecticEuler is https://en.wikipedia.org/wiki/Semi-implicit_Euler_method

Reply:
For some of the descriptions you can just use the one liner from https://docs.sciml.ai/DiffEqDocs/stable/solvers/dynamical_solve/#Symplectic-Integrators